tests/e2e/apps_test.go

```diff
@@ -245,6 +245,40 @@ var _ = Describe("Apps", func() {
 		})
 	})
 
+	Describe("Get app process stats by type", func() {
+		var processStats resourceList[statsResource]
+
+		JustBeforeEach(func() {
+			Eventually(func(g Gomega) {
+				var err error
+				resp, err = adminClient.R().
+					SetResult(&processStats).
+					Get("/v3/apps/" + appGUID + "/processes/web/stats")
+				g.Expect(err).NotTo(HaveOccurred())
+
+				// no 'g.' here - we require all calls to return 200
+				Expect(resp).To(HaveRestyStatusCode(http.StatusOK))
+				g.Expect(processStats.Resources).ToNot(BeEmpty())
+				g.Expect(processStats.Resources[0].Usage).ToNot(BeZero())
+			}).Should(Succeed())
+		})
+
+		BeforeEach(func() {
```

Comment:
Final nitpick, could you move `BeforeEach` above `JustBeforeEach`? I find it more logical.

Reply:
I've changed the order